frontend/public/components/hpa.tsx

```diff
@@ -182,10 +182,12 @@ export const HorizontalPodAutoscalersDetails: React.FC<HorizontalPodAutoscalersD
     <div className="co-m-pane__body">
       <MetricsTable obj={hpa} />
     </div>
-    <div className="co-m-pane__body">
-      <SectionHeading text="Conditions" />
-      <Conditions conditions={hpa.status.conditions} />
-    </div>
+    {hpa?.status?.conditions && (
```

Comment:
Did we break null handling in the `Conditions` component? We should make sure that works since it could break other places, too.